Automattic/wp-calypso
# Themes: Rename "All" tab to "Recommended"

Fixes https://github.com/Automattic/dotcom-forge/issues/1402

#### Proposed Changes

Renames the "All" tab of the Themes page to "Recommended"

Before | After
--- | ---
<img width="987" alt="Screenshot 2023-01-16 at 13 19 56" src="https://user-images.githubusercontent.com/1233880/212677227-70855379-e2d6-43c5-bfdd-880ecd4645cc.png"> | <img width="992" alt="Screenshot 2023-01-16 at 13 19 42" src="https://user-images.githubusercontent.com/1233880/212677250-658294f1-8cac-40e0-bdef-2341df0097bc.png">

An “All” tab label suggests ALL themes will show up here but this tab won’t include all themes available in search.


#### Testing Instructions

- Use the Calypso live link below
- Go to Appearance > Themes
- Make sure the first tab changed to "Recommended"
